volatility3/framework/symbols/windows/extensions/gui.py

```diff
@@ -77,7 +81,7 @@ def desktops(self, symbol_table_name, max_desktops: int = 12):
 class tagDESKTOP(objects.StructType, pool.ExecutiveObject):
     def is_valid(self) -> bool:
         """
-        Enforce a valid sid + owning window station
+        Enforce a valid sid + name
```

Comment:
Does this validate the name?  It looks like the old comment was more accurate (although either way they're both quite brief)...

Reply:
Nice catch. Fixed and expanded.